hbase-server/src/main/java/org/apache/hadoop/hbase/ipc/NettyRpcServer.java

```diff
@@ -125,7 +125,9 @@ protected void initChannel(Channel ch) throws Exception {
             initSSL(pipeline, conf.getBoolean(HBASE_SERVER_NETTY_TLS_SUPPORTPLAINTEXT, true));
           }
           pipeline.addLast(NettyRpcServerPreambleHandler.DECODER_NAME, preambleDecoder)
-            .addLast(createNettyRpcServerPreambleHandler());
+            .addLast(createNettyRpcServerPreambleHandler())
+            .addLast(NettyRpcServerResponseEncoder.HANDLER_NAME,
```

Comment:
Better add some comments to say that why we need to add encoder here

Reply:
@Apache9 , thanks for review，already added some comments.